packages/react-composites/src/composites/CallComposite/adapter/CallAdapter.ts

```diff
@@ -124,18 +124,108 @@ export type CallEndedListener = (event: { callId: string }) => void;
  * @public
  */
 export interface CallAdapterCallManagement {
+  /**
+   * Join the call with microphone initially on/off.
+   *
+   * @param microphoneOn - Whether microphone is initially enabled
+   *
+   * @public
+   */
   joinCall(microphoneOn?: boolean): Call | undefined;
+  /**
+   * Leave the call
+   *
+   * @param forEveryone - Whether to remove all participants when leaving
+   *
+   * @public
+   */
   leaveCall(forEveryone?: boolean): Promise<void>;
+  /**
+   * Start the camera
+   * This method will start rendering a local camera view when the call is not active
+   *
+   * @public
+   */
   startCamera(): Promise<void>;
+  /**
+   * Stop the camera
+   * This method will stop rendering a local camera view when the call is not active
+   *
+   * @public
+   */
   stopCamera(): Promise<void>;
+  /**
+   * Switch between startCamera and stopCamera according to current camera state
+   * This method will stop rendering a local camera view when the call is not active
+   *
+   * @param options - Options to control how video streams are rendered {@link @azure/communication-calling/#VideoStreamOptions }
```

Comment:
Looks like this one is a little bit overlap with start/stop camera method, I would like to remove this one from adapter and use start/stop camera under the hook for implementation, what do you guyz think